test/test262/errors.js

```diff
@@ -0,0 +1,120 @@
+let fs = require('fs');
+let parseModule = require('../../').parseModule;
+let parseScript = require('../../').parseScript;
+let expect = require('expect.js');
+
+let scriptDir = 'node_modules/test262-parser-tests';
+
+function parse(src, isModule, earlyErrors) {
+  (isModule ? parseModule : parseScript)(src, { earlyErrors: earlyErrors });
+}
+
+suite('test262', function () {
+  suite('pass and pass-explicit', function () {
+    let passExcludes = ['995.script.js',   // https://github.com/shapesecurity/shift-parser-js/issues/311
+      '970.script.js',   // This is an invalid test
+      '1012.script.js']; // https://github.com/shapesecurity/shift-parser-js/issues/311
+    fs.readdirSync(scriptDir + '/pass').filter(function (item) {
+      return passExcludes.indexOf(item) === -1;
+    }).forEach(function (f) {
+      let passTree, passExplicitTree;
+      test('does not throw error and generates same tree[' + f + ']', function () {
+        expect(function parsePass() {
+          passTree = parse(
+            fs.readFileSync(scriptDir + '/pass/' + f, 'utf8'),
+                            f.match('.module.js'),
```

Comment:
This indentation. Please not like this. 2 space indentation, always. All arguments should be at the same level.